tools/event-subscriber/pkg/util/logger.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/httputil"
+)
+
+// Logger ...
+func Logger(h http.Handler) http.Handler {
```

Comment:
(nit) The name of the function makes one think that a logger can be created using the function :-D 
```suggestion
func LogHandler(h http.Handler) http.Handler {
```

Reply:
i did not change the code... fixing the event-subscriber should be done in a later refactoring / rewriting issue